.github/workflows/testing.yml

```diff
@@ -140,7 +163,13 @@ jobs:
       # allow all jobs to finish
       fail-fast: false
       matrix:
-        model: ["e3sm", "cesm"]
+        model:
+          - name: e3sm
+            repo: E3SM-Project/E3SM
+            branch: master
+          - name: cesm
+            repo: ESCOMP/CESM
+            branch: cesm3.0-alphabranch
         driver: ["mct", "nuopc"]
```

Comment:
We can probably restructure a little here to avoid this unnecessary loop - e3sm only uses mct and cesm only uses nuopc.